modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -211,6 +212,17 @@
 				6
 				3
 				[/codeblock]
+				To iterate over [float], convert them in the loop.
+				[codeblock]
+				for i in range (3, 0, -1):
+				    print(i / 10.0)
```

Comment:
Technically this also works:

```
	for i in Vector3(0, 5, 0.5):
		print(i)
```

But the approach you wrote here is better, since a float iterator doesn't work reliably for non-perfect floats (anything that isn't a sum of powers of two, so for examply you can't reliably use 0.1 as the step).